scss/utilities/mixins/organisms/_footer.scss

```diff
@@ -0,0 +1,36 @@
+@charset "UTF-8";
+
+//
+// Footer Mixins
+//
+// Mixins for the global footer.
+//
+// Style guide: Mixins.Organisms.Footer
+//
+
+@mixin footer-colors {
+  background-color: $color-cardinal-red;
+  color: $color-white;
+
+  a {
+    color: rgba($color-white, .9);
+    text-decoration: none;
+
+    &:hover {
+      color: $color-white;
+      text-decoration: underline;
+    }
+  }
+}
+
+@mixin footer-wrapper {
```

Comment:
1.  Up until the grid work we had the convention of 1 mixin a file as it followed the convention that bourbon has set out. I would like to keep to that as much as it makes sense.
2. No code comment. 
3. I don't think this mixin is needed. 
4. Name too generic.